runtime/src/main/java/com/tietoevry/quarkus/resteasy/problem/postprocessing/ProblemLogger.java

```diff
@@ -58,11 +55,11 @@ private String serialize(HttpProblem problem) {
     }
 
     private String serializeParameter(Map.Entry<String, Object> param) {
-        try {
-            return param.getKey() + "=" + mapper.writeValueAsString(param.getValue());
-        } catch (JsonProcessingException e) {
-            return param.getKey() + "=" + param.getValue().toString();
+        String serializedValue = param.getValue().toString();
```

Comment:
Not sure but from GitHub PullReq review perspective this smells NPE.

Reply:
True that, will address.